gym/envs/safety/semisuper.py

```diff
@@ -15,13 +15,10 @@
 class SemisuperEnv(gym.Env):
     def step(self, action):
         assert self.action_space.contains(action)
-        self.monitor._before_step(action)
```

Comment:
The safety environments want to provide one reward to the agent (observed_reward) while recording a different reward in the monitor (true_reward). 

(Aside: this would be easy to implement if monitor was a wrapper, since you could have a wrapper that distorts the reward after the monitor wrapper)